lib/omnibus/health_check.rb

```diff
@@ -246,7 +250,7 @@ def run!
       measure("Health check time") do
         log.info(log_key) { "Running health on #{project.name}" }
         bad_libs =  case Ohai["platform"]
-                    when "mac_os_x"
+                    when "macos"
```

Comment:
Not sure of the whole context behind the name change, but perhaps it's worth keeping the old name around?

```suggestion
                    when "macos", "mac_os_x"
```

Reply:
it seems the naming changed following a macos or ruby update (In all honesty I don't know what changed, but the platform name is now `macos` for sure)

Good point about keeping both. There are also other locations that only use the old one which we'll need to update

Reply:
done for this file, I'll prepare another PR to update the remaining references to `mac_os_x`